CHANGELOG.md

```diff
@@ -152,8 +133,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2113**
   - Added Granule not found report to reports endpoint
   - Update reports to return breakdown by Granule of files both in DynamoDB and S3
+- **CUMULUS-2123**
```

Comment:
A lot of these changes are reorganizing the CHANGELOG to be sorted by ticket number ascending